api-reference/v1.0/api/groupsetting_post_groupsettings.md

```diff
@@ -0,0 +1,93 @@
+# Create a group setting
+
+Use this API to create a new setting, based on the templates available in [groupSettingTemplates](../resources/groupsettingtemplate.md). These settings can be at the tenant-level or at an object level (currently only for groups). The creation request must provide [settingValues](../resources/settingvalue.md) for all the settings defined in the template. For group-specific settings, only the setting governing whether members of a group can invite guest users can be set. This will govern this behavior once the ability to add guest users to a group is generally available.
```

Comment:
"These settings can be at the tenant-level or at an object level (currently only for groups)."
Please change to: "These settings can be at the tenant-level or at the group level."  - this is because this will only ever be for Groups (per the name :))